src/Components/ErrorDisplay/ErrorDisplay.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./ErrorDisplay.css"
+import { Link } from 'react-router-dom'
+
+function ErrorDisplay(props) {
+
+    const error = props.errorCode
+    let errorMessage = ''
+
+    if(Number(error) > 500) {
+        errorMessage = 'Please try again later.'
+    }
+    else if(Number(error) > 400) {
+        errorMessage = 'The page you were looking for was not found.'
+    }
+    else{
```

Comment:
Super nit-picky but just add a space after the else here for the sake of consistency :)